koku/cost_models/serializers.py

```diff
@@ -294,6 +389,8 @@ class Meta:
 
     rates = RateSerializer(required=False, many=True)
 
+    # tag_rates = TagRateSerializer(required=False, many=True)
+
```

Comment:
```suggestion
```

Reply:
Whoops! Totally missed this line when I was doing my clean up. I am gonna remove it now.